examples/vault-iam-auth/user-data-auth-client.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# This script is meant to be run in the User Data of each EC2 Instance while it's booting. The script uses the
+# run-consul script to configure and start Consul in client mode. Note that this script assumes it's running in an AMI
+# built from the Packer template in examples/vault-consul-ami/vault-consul.json.
```

Comment:
This comment should probably be updated, as this script does quite a bit more now